airflow/config_templates/default_airflow.cfg

```diff
@@ -581,6 +581,10 @@ authenticate = False
 # DAGs submitted manually in the web UI or with trigger_dag will still run.
 use_job_schedule = True
 
+# Allow externally triggered DagRuns for Execution Dates in the future when True
```

Comment:
Can you add a short documentation describing this parameter?  I am not sure about the place, but I think you can create a new document in the `howto` directory.